dipy/viz/projections.py

```diff
@@ -56,7 +55,8 @@ def sph_project(vertices, val, ax=None, vmin=None, vmax=None, cmap=None,
     --------
     >>> from dipy.data import get_sphere
     >>> verts = get_sphere('symmetric724').vertices
-    >>> ax = sph_project(verts.T, np.random.rand(len(verts.T))) # skip if not has_basemap
+    >>> ax = sph_project(verts.T, np.random.rand(len(verts.T))) # skip
```

Comment:
Sorry - I was wrong. This change is the change that causes the test failures in the bots. Please change this line back. 

This causes Travis to skip this test, when basemap is not present, and the Travis set up doesn't (usually) have this package.